course/src/main/java/in/testpress/course/services/VideoDownloadService.kt

```diff
@@ -83,9 +93,21 @@ class VideoDownloadService : DownloadService(
             Download.STATE_FAILED -> notification = getFailedNotification()
             Download.STATE_REMOVING -> downloadMonitor.deleteVideo(download)
         }
+
+        notification?.contentIntent = getNotifyPendingIntent()
         NotificationUtil.setNotification(this, nextNotificationId, notification)
     }
 
+    private fun getNotifyPendingIntent(): PendingIntent{
+        val notifyIntent = Intent(this, DownloadsActivity::class.java).apply {
```

Comment:
notifyIntent variable does not conveying any meaning, please change it to meaningful name

Reply:
Okay @Karthik-0, will change that.